services/wallet/community/manager.go

```diff
@@ -53,36 +49,7 @@ func (cm *Manager) setCommunityInfo(id string, c *thirdparty.CommunityInfo) (err
 	return cm.db.SetCommunityInfo(id, c)
 }
 
-func (cm *Manager) mustFetchCommunityInfo(communityID string) bool {
```

Comment:
This logic has now been added to the messenger side `func (m *Messenger) FetchCommunity`